lib/provider.js

```diff
@@ -11,76 +11,83 @@ class StorageProvider {
   }
 
   // Open StorageProvider
-  //   callback - function(err, StorageProvider)
-  // eslint-disable-next-line no-unused-vars
-  open(options, callback) {
+  //   callback - <Function>
+  //     err - <Error>
+  //     StorageProvider - <Function>
+  open(options, callback) {  // eslint-disable-line no-unused-vars
     throw new GSError(errorCodes.NOT_IMPLEMENTED);
   }
 
   // Close StorageProvider
-  //   callback - function(err)
-  // eslint-disable-next-line no-unused-vars
-  close(callback) {
+  //   callback - <Function>
+  //     err - <Error>
+  close(callback) {  // eslint-disable-line no-unused-vars
```

Comment:
the length of the line 24 is 84

Reply:
Actually, the length of this line is 58 characters. Though there are other lines that exceed max length.
```
/home/travis/build/metarhia/globalstorage/lib/cursor.js
  79:1  error  Line 79 exceeds the maximum line length of 80  max-len
/home/travis/build/metarhia/globalstorage/lib/provider.js
  71:1  error  Line 71 exceeds the maximum line length of 80  max-len
```
For such case you can put it at the line above using `eslint-disable-next-line no-unused-vars`.